Error compiling to wasm target. Inclusion of getrandom is causing issues.

I don't know how I'm the only one to have this issue, but when compiling with `cw-storage-plus = "1.0.0"` as a regular dependency it is trying to pull in getrandom. I think perhaps getrandom should probably be a dev dependency.

Exact error message is
```
error: the wasm32-unknown-unknown target is not supported by default, you may need to enable the "js" feature. For more information see: https://docs.rs/getrandom/#webassembly-support
   --> /Users/ewoolsey/.cargo/registry/src/github.com-1ecc6299db9ec823/getrandom-0.2.8/src/lib.rs:263:9
    |
263 | /         compile_error!("the wasm32-unknown-unknown target is not supported by \
264 | |                         default, you may need to enable the \"js\" feature. \
265 | |                         For more information see: \
266 | |                         https://docs.rs/getrandom/#webassembly-support");
    | |________________________________________________________________________^

error[E0433]: failed to resolve: use of undeclared crate or module `imp`
   --> /Users/ewoolsey/.cargo/registry/src/github.com-1ecc6299db9ec823/getrandom-0.2.8/src/lib.rs:290:5
    |
290 |     imp::getrandom_inner(dest)
    |     ^^^ use of undeclared crate or module `imp`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `getrandom` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
process exited unsuccessfully: exit status: 101
```

This error only occurs using a wasm target. There is a super hacky workaround which is put this in your toml:
```toml
getrandom = { version = "0.2", features = ["js"] }
cw-storage-plus = "1"
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -46,6 +46,9 @@ jobs:
       - run:
           name: Version information
           command: rustc --version; cargo --version; rustup --version; rustup target list --installed
+      - run:
+          name: Add wasm32 target
+          command: rustup target add wasm32-unknown-unknown && rustup target list --installed
       - restore_cache:
           keys:
             - cargocache-v2-storage-plus:1.64.0-{{ checksum "Cargo.lock" }}
@@ -58,6 +61,12 @@ jobs:
       - run:
           name: Build library for native target (with iterator and macro)
           command: cargo build --locked --all-features
+      - run:
+          name: Build library for Wasm target (default features)
+          command: cargo build --target wasm32-unknown-unknown --locked
+      - run:
+          name: Build library for Wasm target (all features)
+          command: cargo build --target wasm32-unknown-unknown --locked --all-features
       - run:
           name: Run unit tests (with iterator and macro)
           command: cargo test --workspace --locked --all-features

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,12 +28,6 @@ schemars = "0.8.3"
 serde = { version = "1.0", default-features = false, features = ["derive"] }
 cw-storage-macro = { version = "1.0.0", optional = true, path = "macros" }
 
-# We don't use the following dependencies directly. They're dependencies of our dependencies.
-# We specify them to tighten their version requirements so that builds with `-Zminimal-versions` work.
-# Once we bump `cosmwasm-*` deps to a version after `1.1.5`, we can remove these.
-k256 = { version = "0.11.1", features = ["ecdsa"] }
-thiserror = "1.0.13"
-
 [dev-dependencies]
 criterion = { version = "0.3", features = ["html_reports"] }
 # Deactivate default features in order to be able to use this on systems without

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,8 +37,8 @@ rand_xoshiro = { version = "0.6.0", default-features = false }
 
 # We don't use the following dependencies directly. They're dependencies of our dependencies.
 # We specify them to tighten their version requirements so that builds with `-Zminimal-versions` work.
-# Once we bump `cosmwasm-*` deps to a version after `1.1.5`, we can remove these.
-serde_json = "1.0.40"
+# https://github.com/GREsau/schemars/pull/192 is merged, we can update schemars and release this.
+serde_json = "1.0.29"
 
 [[bench]]
 name = "main"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -23,7 +23,7 @@ macro = ["cw-storage-macro"]
 bench = false
 
 [dependencies]
-cosmwasm-std = { version = "1.0", default-features = false }
+cosmwasm-std = { version = "1.1.6", default-features = false }
 schemars = "0.8.3"
 serde = { version = "1.0", default-features = false, features = ["derive"] }
 cw-storage-macro = { version = "1.0.0", optional = true, path = "macros" }

